packages/paste-core/components/alert-dialog/src/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Button} from '@twilio-paste/button';
+import {Stack} from '@twilio-paste/stack';
+import {
+  modalHeaderStyles,
+  modalBodyStyles,
+  modalFooterStyles,
+  ModalDialogOverlay,
+  ModalDialogContent,
+} from '@twilio-paste/modal';
+import {useUID} from '@twilio-paste/uid-library';
+
+export interface AlertDialogProps extends React.HTMLAttributes<HTMLElement> {
+  children: React.ReactNode;
+  isOpen: boolean;
+  heading: string;
+  destructive?: boolean;
+  primaryLabel: string;
+  primaryAction: () => void;
+  secondaryLabel?: string;
+  secondaryAction?: () => void;
+}
+
+export const AlertDialog = React.forwardRef<HTMLDivElement, AlertDialogProps>(
+  (
+    {children, isOpen, heading, destructive, primaryLabel, primaryAction, secondaryLabel, secondaryAction, ...props},
+    ref
+  ) => {
+    const headingID = useUID();
+    const bodyID = useUID();
+
+    const primaryStyle = destructive ? 'destructive' : 'primary';
```

Comment:
Should this be something like `buttonVariant` or `primaryVariant` instead?

Reply:
I like that better, yeah. I'll change it to `primaryVariant`